clinics/manager/manager.go

```diff
@@ -219,6 +224,32 @@ func (c *manager) getDemoPatient(ctx context.Context) (*patients.Patient, error)
 	return patient, nil
 }
 
+func (c *manager) deleteClinic(ctx context.Context, clinicId string) error {
+	filter := patients.Filter{ClinicId: &clinicId}
+	pagination := store.Pagination{Limit: 2}
+	res, err := c.patientsService.List(ctx, &filter, pagination, nil)
+
+	if err != nil {
+		return err
+	}
+	if res == nil {
+		return fmt.Errorf("patient list result not defined")
+	}
+	if !c.patientListAllowsClinicDeletion(res.Patients) {
+		return fmt.Errorf("%w: deletion of non-empty clinics is not allowed", errors.BadRequest)
```

Comment:
Same as above, Is there a reason this "constant" error is wrapped inline and not defined as a variable in the `errors` package?